crontab/notify_old_pp.php

```diff
@@ -88,7 +86,6 @@ function send_pp_reminders($PPer, $projects, $which_message)
 
         $message[] = "==" . _("Are there any problems?") . "==";
         $message[] = sprintf(_("If any project is missing page images or illustrations, please contact the Project Manager (PM) for assistance. If the PM does not respond within a reasonable amount of time, please email %s and request help concerning any problems."), $db_requests_email_addr);
```

Comment:
Add something along the lines of:
`$message[] = sprintf(_("Thank you for volunteering with %s!"), $site_name);`
after the current last para. The old `$site_signoff` included "Thank you". It doesn't, now (which I think is reasonable, because some of the emails we send are better off with customized thank-yous at the end), so should be added both here ...

Reply:
Added.